content/docs/practices/consoles.md

```diff
@@ -9,32 +9,31 @@ It can be tempting to display as much data as possible on a dashboard, especiall
 when a system like Prometheus offers the ability to have such rich
 instrumentation of your applications. This can lead to consoles that are
 impenetrable due to having too much information, that even an expert in the
-system would have difficulty drawing meaning from. Hundreds of graphs on a
-single page isn't unheard of, nor is a hundred plots on a single graph
+system would have difficulty drawing meaning from.
 
 Instead of trying to represent every piece of data you have, for operational
-consoles think of what are the most likely failure modes and how you'd use the
+consoles think of what are the most likely failure modes and how you would use the
 consoles to differentiate them. Take advantage of the structure of your
-services. For example if you've a big tree of services in an online serving
-system, latency in some lower service is a typical problem. You could have one
-big page with every service's information, a better approach is one page per
-service that includes the latency and errors it sees for each service it talks
-to. You can then start at the top and work your way down to the problem
+services. For example, if you have a big tree of services in an online-serving
+system, latency in some lower service is a typical problem. Rather than showing
+every service's information on a single large dashboard, build separate dashboards
+for each service that include the latency and errors for each service they talk
+to. You can then start at the top and work your way down to the problematic
 service.
 
-We've found the following guidelines very effective:
+We have found the following guidelines very effective:
 
 * Have no more than 5 graphs on a console.
 * Have no more than 5 plots (lines) on each graph. You can get away with more if it's a stacked/area graph.
-* If using console templates, try to avoid more than 20-30 entries on the table on the right
+* When using the provided console template examples, try avoiding more than 20-30 entries in the right-hand-side table.
```

Comment:
"try avoiding" doesn't sound right here. It comes across more like walking on the opposite side of the street to keep away from it, rather than not doing it.